examples/onnx/yolov5-videos/README.md

```diff
@@ -0,0 +1,57 @@
+# YoloV5 Detection model
+
+This example deploys a detection model trained using [ultralytics' yolo repo](https://github.com/ultralytics/yolov5) using ONNX.
+We'll use the `yolov5s` model as an example here.
+In can be used to run inference on youtube videos and returns the annotated video with bounding boxes.
+
+The example can be run on both CPU and on GPU hardware.
+
+## Exporting ONNX
+
+To export a custom model from the repo, use the [`model/export.py`](https://github.com/ultralytics/yolov5/blob/master/models/export.py) script.
+The only change we need to make is to change the line
+
+```
+model.model[-1].export = True  # set Detect() layer export=True
+```
+
+to
+
+```
+model.model[-1].export = False
+```
+
+Originally, the ultralytics repo does not export postprocessing steps of the model, e.g. the conversion from the raw CNN outputs to bounding boxes.
+With newer ONNX versions, these can be exported as part of the model making the deployment much easier.
+
+With this modified script, the ONNX graph used for this example has been exported using
+```
+python models/export.py --weights weights/yolov5s.pt --img 416 --batch 1
+```
+
+
+## Sample Prediction
```

Comment:
Since people will probably try to use the example with the pre-exported model before exporting their own, how do you feel about switching the order of this and the previous section?

Reply:
Nice catch! Fully agree